README.rst

```diff
@@ -53,27 +53,37 @@ From source:
     python setup.py install
 
 
+Why Ignite?
+===========
+Ignite's high level of abstraction assumes less about the type of network (or networks) that you are training, and we require the user to define the closure to be run in the training and validation loop. This level of abstraction allows for a great deal more of flexibility, such as co-training multiple models (i.e. GANs) and computing/tracking multiple losses and metrics in your training loop.
+
+Ignite also allows for multiple handlers to be attached to events, and a finer granularity of events in the engine loop.
+
+
 Documentation
 =============
 API documentation and an overview of the library can be found `here <https://pytorch.org/ignite/index.html>`_.
 
+
+Structure
+=========
+- **ignite**: Core of the library, contains supervised trainer and evaluator, all of the classic machine learning metrics and a variety of handlers to ease the pain of training neural networks! 
```

Comment:
Some nits:
```suggestion
- **ignite**: Core of the library, contains supervised trainer and evaluator, all of the classic machine learning metrics and a variety of handlers to ease the pain of training and validation of neural networks! 
```

Reply:
should we remove the `supervised trainer` part and just say an engine for training and evaluating

Reply:
Incorporated!